### PR TITLE
reddit slideshow.

### PR DESCRIPTION
**Pull request by evans760**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 2

Completeness Level (1-5):1

What did you think of this deliverable?:
really challenging for me.
I didn't get too far but, i'll continue to work on this project.

### CODE REVIEW

**Comment by scottheron** [comment]
Good start Thomas, keep going, check out what others have done to help you along. I'll keep looking at it and see if I can help you get further on with it. Brian might be able to give you some feedback too.

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,26 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Hello Front-End</title>
+  <title>PhotoSlideShow</title>
+
+  <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+  <script src="ajax.js"></script>
```
Comment (bhague1281) [inline src/index.html:9]: You'll want the script tags right before the `</body>` tag. Also, `ajax.js` doesn't exist. You'll want to include the source for `js/slide.js`

File: src/js/slide.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  //get the users search input and save it in a variable.
+  //use the input placeholder value (like "kittens") as default value
+  var input = $("#query");
+  var useQuery = input.val() || input.attr("placeholder");
+
+  console.log("searching for:", useQuery);
+
+  $.get('https://www.reddit.com/search.json', {
+    q: userQuery
```
Comment (bhague1281) [inline src/js/slide.js:41]: This variable doesn't match the one you defined above, so it'll throw an error for an undefined value.

File: src/index.html [hunk 3]
```diff
@@ -3,9 +3,26 @@
...
+
+
 <body>
-  <h1>Hello Front-End</h1>
-</body>
+  <h1>Photo Slide Show</h1>
+
+<form id="search-form">
+
+    <input id="qurey" placeholder="kittens">
```
Comment (bhague1281) [inline src/index.html:18]: This id is spelled wrong, and doesn't match the ID you're selecting in your script.

**Comment by bhague1281** [comment]
Good start. I'd recommend doing the following:
- Work on including the correct script file on the page. It's really hard to write code when you can't test it.
- Double-check the spellings of your variables and functions. I noted a couple of misspellings, but there were a few more that I didn't note of. Again, test your code, and you'll usually get an error when something is misspelled.

**Pull request closed without merging**